### PR TITLE
[MB-10155] Added domestic NTS packing to service item calculations

### PR DESCRIPTION
**Pull request by reggieriser**
## Summary

This PR adds domestic NTS packing (`DNPK`) support to the service item calculations in the TIO interface.

Note: There appears to be an issue with the horizontal width in the "large" view (see screenshot below) that causes the right-most side to get cut off and forcing scrolling.  That seems to be an issue generally, though, with the component because I see it happening with domestic linehaul (`DLH`) as well and probably for any calculations that have 5 columns of data like `DNPK` and `DLH` do.  Glad to fix it here if possible, but may also be worth a separate story.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

If you don't already have one handy, you'll need to create an NTS shipment and run it all the way through to payment request creation (make sure you include NTS packing). Those steps are detailed in #7849.

After doing that, login as a TIO and you should be able to see the service item calculations in two places:
- when approving service items (you'll see the "small" table of calculations)
- when viewing the approved service items (you'll see the "large" table of calculations)

You can also see `DNPK` in storybook by going to the `ServiceItemCalculations` component and choosing `DNPK`. 

## Verification Steps for Author

These are to be checked by the author.

- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots

"Small" view:
Screen Shot 2021-12-16 at 4 03 52 PM

"Large" view:
Screen Shot 2021-12-16 at 4 05 55 PM

### CODE REVIEW

File: src/components/Office/ServiceItemCalculations/helpers.test.js [hunk 1]
```diff
@@ -497,7 +497,7 @@ describe('makeCalculations', () => {
     });
 
     it('returns the correct data for mileage below 50 with matching ZIP3s', () => {
-      const result = makeCalculations('DDDSIT', 99999, testParams.DomesticDestinationSITDeliveryMachingZip3);
+      const result = makeCalculations('DDDSIT', 99999, testParams.DomesticDestinationSITDeliveryMatchingZip3);
```
Comment (reggieriser) [inline src/components/Office/ServiceItemCalculations/helpers.test.js:500]: Just fixing an existing typo I noticed in this identifier.

**Comment by amanda5iu** [comment]
> Note: There appears to be an issue with the horizontal width in the "large" view (see screenshot below) that causes the right-most side to get cut off and forcing scrolling. That seems to be an issue generally, though, with the component because I see it happening with domestic linehaul (DLH) as well and probably for any calculations that have 5 columns of data like DNPK and DLH do. Glad to fix it here if possible, but may also be worth a separate story.

That's intentional - it's meant to accommodate service items that have a lot of pricing data to display. We could make the columns narrower, but at a certain point that starts to impact readability. The horizontal scrolling admittedly isn't ideal - maybe something to revisit post-launch.

**Review by amanda5iu — APPROVED** [review]
Looks great!

**Review by amanda5iu — APPROVED** [review]

**Review by donaldthai — APPROVED** [review]
LGTM 🚀

**Comment by donaldthai** [comment]
@Mergifyio rebase

**Pull request merged**